src/components/filters/RangeFilter.vue

```diff
@@ -76,19 +100,29 @@ export default Vue.extend({
   },
   watch: {
     value (newValue) {
-      this.sliderValue[0] = Math.min(newValue[0], newValue[1])
-      this.sliderValue[1] = Math.max(newValue[0], newValue[1])
+      if (newValue[0] == null && newValue[1] == null) {
+        this.sliderValue = [null, null]
+      }
     }
   },
   methods: {
+    setSliderValue (id, value) {
+      this.sliderValue[id] = value
+      this.handleChange()
+    },
     handleChange () {
-      if (this.sliderValue[0] === this.min && this.sliderValue[1] === this.max) {
-        this.$emit('input', undefined)
-      } else {
-        this.sliderValue = [parseFloat(Math.max(this.min, this.sliderValue[0])), parseFloat(Math.min(this.max, this.sliderValue[1]))]
-        // clone to break reactive loop
-        this.$emit('input', [...this.sliderValue])
+      let minVal = null
+      let maxVal = null
+      if (this.sliderValue[0] != null) {
+        minVal = parseFloat(Math.min(Math.max(this.sliderValue[0], this.min), this.max))
```

Comment:
2 time .max ? is that what we want, maybe add a comment on why this is , not clear to me by looking at the code, but that could just be me ;)